### PR TITLE
PYIC-3313: Separate Feature Flag to enable bundling of the CIMIT VC

### PR DESCRIPTION
**Pull request by TalatAydinGDS**
### What changed

Separate Feature Flag to enable bundling of the CIMIT VC

### Why did it change

Need to decouple switchover to CIMIT VC from bundling CIMIT VC in the user identity bundle.

### Issue tracking

- [PYIC-3313](https://govukverify.atlassian.net/browse/PYIC-3313)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

[PYIC-3313]: https://govukverify.atlassian.net/browse/PYIC-3313?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: lambdas/build-user-identity/src/test/java/uk/gov/di/ipv/core/builduseridentity/BuildUserIdentityHandlerTest.java [hunk 1]
```diff
@@ -246,6 +248,53 @@ void shouldReturnCredentialsWithCiMitVCOnSuccessfulUserInfoRequest()
         verify(mockCiMitService, times(1)).getContraIndicatorsVCJwt(any(), any(), any());
     }
 
+    @Test
+    void
+            shouldReturnCredentialsWithCiMitVCOnSuccessfulUserInfoRequestWhenBundleCimitVCFeatureFlagIsNotEnable()
```
Comment (therealslimsladey) [inline lambdas/build-user-identity/src/test/java/uk/gov/di/ipv/core/builduseridentity/BuildUserIdentityHandlerTest.java]: `shouldReturnCredentialsWithCiMitVCOnSuccessfulUserInfoRequestWhenBundleCimitVCFeatureFlagIsNotEnable` should be `shouldReturnCredentialsWithoutCiMitVCOnSuccessfulUserInfoRequestWhenBundleCimitVCFeatureFlagIsNotEnabled`?

**Review by therealslimsladey — CHANGES_REQUESTED** [review]
Looks good. Just the one test method that needs renaming, I think.

File: lambdas/build-user-identity/src/test/java/uk/gov/di/ipv/core/builduseridentity/BuildUserIdentityHandlerTest.java (hunk 1 above)
Comment (TalatAydinGDS) [reply lambdas/build-user-identity/src/test/java/uk/gov/di/ipv/core/builduseridentity/BuildUserIdentityHandlerTest.java]: Done

**Review by therealslimsladey — APPROVED** [review]
lgtm

**Pull request merged**